nylas/client/restful_models.py

```diff
@@ -83,6 +93,12 @@ class Message(NylasAPIObject):
              "account_id", "object", "snippet", "starred", "subject",
              "thread_id", "to", "unread", "starred", "_folder", "_labels",
              "headers"]
+    datetime_attrs = {
+        "received_at": "date",
+        # The following are used for filtering:
```

Comment:
This is a little bit confusing. Would it make sense to have a separate class field just for converting filter-only attributes? E.g. something like datetime_filter_attrs?